Bottom: 2c30e2bfaf78fecc010c200d5d8789f81a5a9f91
Top:    8a716d8795b3039fb32553d82fdee2a611a28fb4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:53:53 +0000

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 720543f..0f67e51 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -673,6 +673,9 @@ object3d_from_board_outline (void)
 
   board_outline = board_outline_poly (true);
 
+  if (board_outline == NULL)
+    return NULL;
+
   /* Loop over all board outline pieces */
   pa = board_outline;
   do {
